### PR TITLE
Show cryptocurrency info, implement RDFA schema

### PR DESCRIPTION
**Pull request by DianaCrainic**

### CODE REVIEW

File: cryk-ui/src/pages/CryptoInformation.tsx [hunk 1]
```diff
@@ -61,36 +62,56 @@ export default function CryptoInformation(props: any) {
...
             <div>
                 <Helmet>
                     <title>{title}</title>
                 </Helmet>
-                <div className="page-container">
-                    <div className="title">
-                        <h1>{cryptocurrency?.symbol}</h1>
+                <div className="page-container" vocab="http://purl.org/net/bel-epa/doacc#" typeof="Cryptocurrency">
+                    <div className="title" property="http://purl.org/net/bel-epa/doacc#title" 
+                    about="http://purl.org/net/bel-epa/doacc#{id}">
```
Comment (IoanSava) [inline cryk-ui/src/pages/CryptoInformation.tsx]: This results in: `about="http://purl.org/net/bel-epa/doacc#{id}"` but it should be `about="http://purl.org/net/bel-epa/doacc#D0b6c8010-821a-42ef-8975-950ebb3673b7"` (for example). You should use string interpolation

File: cryk-ui/src/pages/CryptoInformation.tsx [hunk 2]
```diff
@@ -150,26 +151,47 @@ export default function CryptoInformation(props: any) {
...
+                        <h1 property="http://purl.org/net/bel-epa/doacc#symbol">{symbol}</h1>
                     </div>
-                    <p className="crypto-description">
-                        {cryptocurrency?.description}
+                    <p className="crypto-description" property="http://purl.org/net/bel-epa/doacc#description"
```
Comment (IoanSava) [inline cryk-ui/src/pages/CryptoInformation.tsx]: Read this document and update types and properties for each attribute (website, source, etc.): https://doacc.github.io/concepts/doacc-owl/index.html

File: cryk-ui/src/pages/CryptoInformation.tsx [hunk 3]
```diff
@@ -150,26 +151,47 @@ export default function CryptoInformation(props: any) {
...
+                        typeof="http://purl.org/net/bel-epa/doacc#description">
+                        {description}
+                    </p>
+                    <p className="crypto-website" property="http://purl.org/net/bel-epa/doacc#website"
+                        typeof="http://purl.org/net/bel-epa/doacc#WebSite">
+                        <span property="http://purl.org/net/bel-epa/doacc#name">Official website: </span>
```
Comment (IoanSava) [inline cryk-ui/src/pages/CryptoInformation.tsx]: The doacc vocab does not contain http://purl.org/net/bel-epa/doacc#name, http://purl.org/net/bel-epa/doacc#url or http://purl.org/net/bel-epa/doacc#value

File: cryk-ui/src/pages/Cryptocurrencies.tsx [hunk 4]
```diff
@@ -130,21 +130,23 @@ export default function Cryptocurrencies() {
                 <Helmet>
                     <title>{title}</title>
                 </Helmet>
-                <div className="page-container">
+                <div className="page-container" vocab="http://purl.org/net/bel-epa/doacc#" typeof="Cryptocurrency">
```
Comment (IoanSava) [inline cryk-ui/src/pages/Cryptocurrencies.tsx]: Remove this

File: cryk-ui/src/pages/Cryptocurrencies.tsx [hunk 5]
```diff
@@ -130,21 +130,23 @@ export default function Cryptocurrencies() {
...
                     <div className="title">
-                        <h1>Cryptocurrency Knowledge Manager</h1>
+                        <h1 property="http://purl.org/net/bel-epa/doacc#name">Cryptocurrency Knowledge Manager</h1>
```
Comment (IoanSava) [inline cryk-ui/src/pages/Cryptocurrencies.tsx]: Remove this

File: cryk-ui/src/pages/Cryptocurrencies.tsx [hunk 6]
```diff
@@ -130,21 +130,23 @@ export default function Cryptocurrencies() {
...
                     </div>
                     <div className="buttons-container">
                         <Button
                             className="visualizations-button"
                             variant="outlined"
                             size="large"
                             onClick={() => { navigate(`/cryptos/visualizations`) }}
+                            property="http://purl.org/net/bel-epa/doacc#visualizations"
```
Comment (IoanSava) [inline cryk-ui/src/pages/Cryptocurrencies.tsx]: Remove this

File: cryk-ui/src/pages/Cryptocurrencies.tsx [hunk 7]
```diff
@@ -130,21 +130,23 @@ export default function Cryptocurrencies() {
...
                         >
                             Visualizations
                         </Button>
                     </div>
-                    <div className="cards-container">
+                    <div className="cards-container" property="http://purl.org/net/bel-epa/doacc#cryptocurrency"
```
Comment (IoanSava) [inline cryk-ui/src/pages/Cryptocurrencies.tsx]: Remove this. We should add rdfa within `CryptoCard` component

File: cryk-ui/src/pages/Cryptocurrencies.tsx [hunk 8]
```diff
@@ -130,21 +130,23 @@ export default function Cryptocurrencies() {
...
+                        typeof="http://purl.org/net/bel-epa/doacc#Cryptocurrency">
```
Comment (IoanSava) [inline cryk-ui/src/pages/Cryptocurrencies.tsx]: Remove this

File: cryk-ui/src/pages/Cryptocurrencies.tsx [hunk 9]
```diff
@@ -155,15 +157,16 @@ export default function Cryptocurrencies() {
                         size="large"
                         page={currentPage}
                         variant="outlined"
-                        onChange={(event, value) => setCurrentPage(value)} />
+                        onChange={(event, value) => setCurrentPage(value)}
+                        property="http://purl.org/net/bel-epa/doacc#pagination" />
```
Comment (IoanSava) [inline cryk-ui/src/pages/Cryptocurrencies.tsx]: Remove this

File: cryk-ui/src/pages/Cryptocurrencies.tsx [hunk 10]
```diff
@@ -155,15 +157,16 @@ export default function Cryptocurrencies() {
...
                     {totalCoinsStats.length > 0 &&
                         <>
-                            <h2>Total coins</h2>
+                            <h2 property="http://purl.org/net/bel-epa/doacc#totalCoins">Total coins</h2>
```
Comment (IoanSava) [inline cryk-ui/src/pages/Cryptocurrencies.tsx]: Remove this

File: cryk-ui/src/pages/Cryptocurrencies.tsx [hunk 11]
```diff
@@ -155,15 +157,16 @@ export default function Cryptocurrencies() {
...
                             {getBarChart(totalCoinsStats)}
                         </>}
                     {blockTimeStats.length > 0 &&
                         <>
-                            <h2>Block time</h2>
+                            <h2 property="http://purl.org/net/bel-epa/doacc#totalCoins">Block time</h2>
```
Comment (IoanSava) [inline cryk-ui/src/pages/Cryptocurrencies.tsx]: Remove this

File: cryk-ui/src/pages/CryptocurrenciesVisualizations.tsx [hunk 12]
```diff
@@ -106,18 +106,19 @@ export default function CryptocurrenciesVisualizations() {
                 <Helmet>
                     <title>{title}</title>
                 </Helmet>
-                <div className="page-container">
+                <div className="page-container" vocab="http://purl.org/net/bel-epa/doacc#" typeof="Visualization">
```
Comment (IoanSava) [inline cryk-ui/src/pages/CryptocurrenciesVisualizations.tsx]: Remove this

File: cryk-ui/src/pages/CryptocurrenciesVisualizations.tsx [hunk 13]
```diff
@@ -106,18 +106,19 @@ export default function CryptocurrenciesVisualizations() {
...
                     <div className="title">
-                        <h1>Visualizations</h1>
+                        <h1 property="http://purl.org/net/bel-epa/doacc#name">Visualizations</h1>
```
Comment (IoanSava) [inline cryk-ui/src/pages/CryptocurrenciesVisualizations.tsx]: Remove this

File: cryk-ui/src/pages/CryptocurrenciesVisualizations.tsx [hunk 14]
```diff
@@ -106,18 +106,19 @@ export default function CryptocurrenciesVisualizations() {
...
                     </div>
-                    <h2>Number of cryptocurrencies by protection scheme</h2>
+                    <h2 property="http://purl.org/net/bel-epa/doacc#name">Number of cryptocurrencies by protection scheme</h2>
```
Comment (IoanSava) [inline cryk-ui/src/pages/CryptocurrenciesVisualizations.tsx]: Remove this

File: cryk-ui/src/pages/CryptocurrenciesVisualizations.tsx [hunk 15]
```diff
@@ -128,7 +129,7 @@ export default function CryptocurrenciesVisualizations() {
                         />
                         <Tooltip />
                     </PieChart>
-                    <h2>Number of cryptocurrencies by founded date</h2>
+                    <h2 property="http://purl.org/net/bel-epa/doacc#name">Number of cryptocurrencies by founded date</h2>
```
Comment (IoanSava) [inline cryk-ui/src/pages/CryptocurrenciesVisualizations.tsx]: Remove this

File: cryk-ui/src/pages/CryptoInformation.tsx [hunk 16]
```diff
@@ -150,26 +151,47 @@ export default function CryptoInformation(props: any) {
...
         <HelmetProvider>
             <div>
                 <Helmet>
                     <title>{title}</title>
                 </Helmet>
-                <div className="page-container">
-                    <div className="title">
-                        <h1>{cryptocurrency?.symbol}</h1>
+                <div className="page-container" vocab="http://purl.org/net/bel-epa/doacc#" typeof="Cryptocurrency">
+                    <div className="title" property="http://purl.org/net/bel-epa/doacc#title"
```
Comment (IoanSava) [inline cryk-ui/src/pages/CryptoInformation.tsx]: http://purl.org/net/bel-epa/doacc#title does not exist in doacc vocab

**Review by CipDan — APPROVED** [review]

**Review by IoanSava — APPROVED** [review]

**Pull request merged**